### PR TITLE
Fix unable to select items in a multi-picklist #953

### PR DESCRIPTION
**Pull request by MortimerGoro**

### CODE REVIEW

File: app/src/common/shared/org/mozilla/vrbrowser/browser/SessionStore.java [hunk 1]
```diff
@@ -497,6 +497,9 @@ public void setRegion(String aRegion) {
     }
 
     public String getHomeUri() {
+        if (true) {
+            return "http://192.168.0.12:8000/";
+        }
```
[the remark this answers is not in the document]
Comment (MortimerGoro) [reply app/src/common/shared/org/mozilla/vrbrowser/browser/SessionStore.java]: Fixed

File: app/src/common/shared/org/mozilla/vrbrowser/input/MotionEventGenerator.java [hunk 2]
```diff
@@ -126,6 +126,7 @@ public static void dispatchScroll(Widget aWidget, int aDevice, float aX, float a
         device.mPreviousWidget = aWidget;
         device.mCoords[0].setAxisValue(MotionEvent.AXIS_VSCROLL, aY);
         device.mCoords[0].setAxisValue(MotionEvent.AXIS_HSCROLL, aX);
+        //generateEvent(aWidget, device, MotionEvent.ACTION_HOVER_MOVE, true);
```
[the remark this answers is not in the document]
Comment (MortimerGoro) [reply app/src/common/shared/org/mozilla/vrbrowser/input/MotionEventGenerator.java]: Fixed. I had to do a lot of research to fix the scroll issue, I thought it was related to fake touch events. Forgot to clean this file

**Comment by cvan** [comment]
Test case to use (the third field down):

- https://webxr.sh/forms, select `#22`
- https://mdn.github.io/learning-area/html/forms/native-form-widgets/drop-down-content.html

**Comment by MortimerGoro** [comment]
@cvan can you reproduce the highlight issue consistently?  I tried many times and only was able to reproduce it once. At the same time https://github.com/MozillaReality/FirefoxReality/issues/1007 happened too, but not on the others. So I think they are related and it´s a GV bug, because the values in the prompt are kept selected correctly.

File: app/src/common/shared/org/mozilla/vrbrowser/ui/widgets/prompts/ChoicePromptWidget.java [hunk 3]
```diff
@@ -165,6 +166,10 @@ public void setChoices(Choice[] choices) {
         mListItems = getWrappedChoices(choices);
         mAdapter = new ChoiceAdapter(getContext(), R.layout.prompt_choice_item, mListItems);
         mList.setAdapter(mAdapter);
+        int height = WidgetPlacement.dpDimension(getContext(), R.dimen.prompt_choice_min_height);
+        height += choices.length * 20;
```
[the remark this answers is not in the document]
Comment (MortimerGoro) [reply app/src/common/shared/org/mozilla/vrbrowser/ui/widgets/prompts/ChoicePromptWidget.java]: Done

**Comment by bluemarvin** [comment]
@cvan trying to roll a release. Any additional issues can be handled in follow up.

**Comment by cvan** [comment]
> @cvan can you reproduce the highlight issue consistently? I tried many times and only was able to reproduce it once. At the same time #1007 happened too, but not on the others attemps. So I think they are related and it's a GV bug, because the values in the prompt are kept selected correctly.

I can consistently reproduce the highlight issue with `<option selected>` in `<select multiple>` issue and (https://github.com/MozillaReality/FirefoxReality/issues/1007#issuecomment-473084101). I've filed issue #1013.

I can also consistently reproduce (https://github.com/MozillaReality/FirefoxReality/issues/1007#issuecomment-473084101) the bug with the hidden text in `<option disabled>`. thanks for filing [GeckoView bug 1535275](https://bugzilla.mozilla.org/show_bug.cgi?id=1535275) to track.

**Pull request merged**